libraries/core_libs/network/src/tarcap/packets_handlers/dag_sync_packet_handler.cpp

```diff
@@ -102,18 +101,26 @@ void DagSyncPacketHandler::process(const PacketData& packet_data, const std::sha
 
     received_dag_blocks_str += block.getHash().abridged() + " ";
 
-    auto status = checkDagBlockValidation(block);
-    if (!status.first) {
-      // This should only happen with a malicious node or a fork
-      std::ostringstream err_msg;
-      err_msg << "DagBlock " << block.getHash() << " validation failed: " << status.second;
+    if (block.getLevel() > peer->dag_level_) peer->dag_level_ = block.getLevel();
```

Comment:
I guess we should set this after the verification, as when verification fails we will disconnect

Reply:
Done